lightning_pose/data/datasets.py

```diff
@@ -214,6 +194,8 @@ def __getitem__(self, idx: int):
                 transformed_images = np.asarray(images)
                 transformed_keypoints = keypoints_on_image.numpy().reshape(-1)
 
+            # send frames to tensors and normalize
+            # need to loop through because ToTensor transform only operates on single images
             for i, transformed_image in enumerate(transformed_images):
                 transformed_image = self.pytorch_transform(transformed_image)
                 if i == 0:
```

Comment:
I'm not sure why we can't take out 
```
image_frames_tensor = torch.unsqueeze(transformed_image, dim=0)
```
outside of the loop? probably there's a reason

Reply:
yeah i tried this, unfortunately the pytorch transforms.Normalize() only works on single images for some reason; maybe we can revisit this in the future, seems like we shouldn't have to loop